Src/WitsmlExplorer.Api/Workers/ImportLogDataWorker.cs

```diff
@@ -55,7 +55,24 @@ public override async Task<(WorkerResult, RefreshAction)> Execute(ImportLogDataJ
             }
 
             //Todo: find a way to determine the maximum amount of rows that can be sent to the WITSML server then pass that amount to the CreateImportQueries method
-            WitsmlLogs[] queries = CreateImportQueries(job, chunkSize).ToArray();
+            var ro = job.DataRows
```

Comment:
Rename to something like _dataRows_ to make it easier to read than just _ro_.

Reply:
done